src/components/SQFormTransferTool/SQFormTransferProductPanels.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Box} from '@mui/material';
+import {Accordion} from 'scplus-shared-components';
+import type {AccordionPanelType} from 'scplus-shared-components';
+import type {TransferProduct} from './types';
+
+export type SQFormTransferProductPanelsProps = {
+  transferProducts: TransferProduct[];
+};
+
+function getPanel(
+  transferProducts: Array<TransferProduct>
+): AccordionPanelType[] | null {
+  if (!transferProducts?.length) {
+    return [];
+  }
+
+  return transferProducts.map(({productID, productDisplayName, enabled}) => {
+    return {
+      body: <>TO BE IMPLEMENTED SC3-1809 (accordions) and SC3-1810 (steps)</>,
+      name: `${productID}`,
+      title: productDisplayName,
+      isDisabled: !enabled,
+    };
+  });
+}
+
+export default function SQFormTransferProductPanels({
+  transferProducts,
+}: SQFormTransferProductPanelsProps): React.ReactElement | null {
+  const panels = getPanel(transferProducts) ?? [];
+  if (!panels.length) {
+    return null;
```

Comment:
Is this possible? An edge case? Or an indication of an error? If this is unexpected but should still be handled maybe we can throw a `console.warn` in here?

Reply:
The TODO comment below this, is to help circle back to this very question. The requirements have not yet defined if we really want a loading indicator or what do to if we fail to load. 

I will resolve this once I can get better requirements.